packages/eslint-plugin-wdio/lib/utils/helpers.ts

```diff
@@ -0,0 +1,14 @@
+export const isCommand = function(expression: any, command: 'pause' | 'debug'): boolean {
+    const callee = expression?.callee
+
+    if (
+        callee &&
+        callee?.object?.name === 'browser' &&
+        callee?.property?.name === command
+    ) {
+        return true
+    }
+
+    return false
```

Comment:
```suggestion
    return (
        callee &&
        callee?.object?.name === 'browser' &&
        callee?.property?.name === command
    )
```

Reply:
Ah good call, I committed an update.

Thanks for looking at this.